psi4/src/psi4/dfocc/manager.cc

```diff
@@ -818,9 +843,18 @@ void DFOCC::ccsd_manager() {
     Process::environment.globals["CURRENT CORRELATION ENERGY"] = Eccsd - Escf;
     Process::environment.globals["CCSD TOTAL ENERGY"] = Eccsd;
     Process::environment.globals["CCSD CORRELATION ENERGY"] = Eccsd - Escf;
+    variables_["CURRENT CORRELATION ENERGY"] = Eccsd - Escf;
+    variables_["CCSD TOTAL ENERGY"] = Eccsd;
+    variables_["CCSD CORRELATION ENERGY"] = Eccsd - Escf;
+    if ((reference == "RHF") or (reference == "UHF")) {
+        Process::environment.globals["CCSD DOUBLES ENERGY"] = Eccsd - Escf;
+        Process::environment.globals["CCSD SINGLES ENERGY"] = 0.0;
+        variables_["CCSD DOUBLES ENERGY"] = Eccsd - Escf;
+        variables_["CCSD SINGLES ENERGY"] = 0.0;
```

Comment:
I think so. Singles are only nonzero for ROHF, so singles = 0 and doubles = ccsd correlation seems sound. Am I overlooking something?

Reply:
For reference, see eq. 134 of the Crawford/Schaefer CC paper. The CC energy has a T1 term which vanishes for the unrestricted SCF orbitals, a product-of-T1 term, and a T2 term. I would assume the product-of-T1 term would also count as singles energy, and it does not vanish for unrestricted SCF orbitals. It may be in doubles for historical reasons, though.

Reply:
A question of convention. A doubles _energy_ seems to be avoided in modern literature. If the singles energy is the expectation value of T1 (eq.128, from above), then perhaps the doubles energy is the expectation value of T2 (eq.130)?  Since the CC ansatz couples the excitation operators, perhaps the attempt to separate a 'doubles' energy is too ambiguous.  I don't know, but there are enough experts around I believe.